src/pool/srv_pool.c

```diff
@@ -1553,6 +1565,13 @@ pool_svc_lookup(uuid_t uuid, struct pool_svc **svcp)
 	d_iov_t	id;
 	int		rc;
 
+	rc = ds_pool_failed_lookup(uuid);
+	if (rc) {
+		D_ERROR(DF_UUID": failed to start: "DF_RC"\n",
+			DP_UUID(uuid), DP_RC(rc));
+		return -DER_NO_SERVICE;
```

Comment:
Yup, originally i thought about return this error directly to users, but this might be not good as well, eg pool might fail to up because of out of memory etc. pool query later return out of memory might be confusing as well.

So instead i thought return "no service available" might be a good choice in generally, but i'd like to keep psf_error, at least to output this error number in the log.